documentation/book/proc-upgrading-brokers-newer-kafka.adoc

```diff
@@ -16,14 +16,24 @@ This procedure describes how to upgrade a {ProductName} Kafka cluster from one v
 
 .Procedure
 
-. Consult the table below and determine whether the new Kafka version has a different log message format version than the previous version. 
+. Consult the table below and determine whether the new Kafka version has a different log message format version than the previous version.
 +
+ifdef::StrimziUpgrades[]
 include::snip-kafka-versions.adoc[leveloffset=+1]
+endif::StrimziUpgrades[]
+ifndef::StrimziUpgrades[]
+[options="header"]
+|=======================
+|Kafka version |Interbroker protocol version  |Log message format version | Zookeeper version
+|2.0.0         |2.0                           |2.0                        | 3.4.13
+|2.1.1         |2.1                           |2.1                        | 3.4.13
+|=======================
+endif::StrimziUpgrades[]
```

Comment:
Wouldn't it be far simpler to just change the `kafka-versions` file before generating the documentation? That should cause the generated table to be correct. This is, at least, how I imagined we would do it when I added that automation, so if that doesn't work for docs for some reason I'd like to know, otherwise the benefits of automating this stuff aren't really so beneficial.